CMakeLists.txt

```diff
@@ -250,7 +250,10 @@ if(csocket_files STREQUAL "")
 	if(git_status_var)
 		message(FATAL_ERROR
 			" It looks like git submodules are not initialized.\n"
-			" Either this is not a git clone, or you don't have git installed")
+			" Either this is not a git clone, or you don't have git installed. \n"
+			" If you downloaded the tarball or zip from git, third_party modules are not included. \n"
```

Comment:
1. submodules.
ZNC modules are something completely different.

2. github
How do you download it from "git"?

Reply:
1: https://github.com/znc/znc/tree/master/third_party as in "third_party modules". Would `external dependencies` be better wording? Suggestions?

2: Front-page of project, `Clone or Download` and you can download as zip. 
Or from [releases page on GitHub](https://github.com/znc/znc/releases)

Reply:
1. I'm not sure what do you mean. "External dependencies" are more external than this, e.g. openssl is external, and we don't ship it together with ZNC, even in the actual release tarballs. In normal ZNC distribution third_party/ is supposed to be filled with 2 particular libraries, but in the downloaded .zip, it's empty. "git submodule" is the correct term here. If your concern is that "git submodule" is confusing, I don't know what to advise then.

2. Sure. That's specific to GitHub though, not Git. I'm sad people these days are confusing git and github...

Reply:
2. Moreover, if you use `git clone` instead of Github UI, the build works and/or tells you how exactly to get the submodules using `git` command.

Reply:
1. That's why I asked what to call them. Sure they are git submodules that you include into your code (external repo). Most of your users probably don't know what `git submodules` are, but they are probably not the ones to compile from source either on the other hand.
Anyways, let's just encourage them to leech the tarball from the website.


2. Actuall it's not only GitHub, most of the versioning control systems with a WebUI have this feature like GitLab, BitBucket, Stash, Azure DevOps and more.. but that's not to focus on here. 

Let's just tell the users submodules are not initialized, if they know what to do wit that, good. 
If they don't know what to do, encourage them to leech the tarball from znc website.


Are you happy with that?

Reply:
2. Do they also silently fail to include submodules in the resulting zip file?